tests/services/orders/order_store_service/test_microbial_store_order_service.py

```diff
@@ -86,15 +84,8 @@ def test_store_microbial_samples(
     assert base_store.get_all_organisms().count() == 1
 
     # WHEN storing the order
-    new_samples = store_microbial_order_service.store_items_in_status(
-        customer_id=microbial_status_data["customer"],
-        order=microbial_status_data["order"],
-        ordered=dt.datetime.now(),
-        ticket_id=ticket_id,
-        items=microbial_status_data["samples"],
-        comment="",
-        data_analysis=Workflow.MICROSALT,
-        data_delivery=DataDelivery.FASTQ_QC,
+    new_samples = store_microbial_order_service.store_order_data_in_status_db(
+        order=microbial_status_data["order"]
```

Comment:
I think you should manually create a fixture for a valid microbial order and use it here instead, as this picture uses deprecated functions from the store service. (Not sure how this test is passing 😅 )

Reply:
fix this test https://github.com/Clinical-Genomics/cg/blob/cbeb28c5e2981e2d22b4f578145f1d882818653a/tests/services/orders/order_store_service/test_microbial_store_order_service.py#L17